src/firefly/js/drawingLayers/ActiveTarget.js

```diff
@@ -27,7 +27,7 @@ var idCnt=0;
 
 
 function creator(initPayload, presetDefaults) {
-    var drawingDef= makeDrawingDef('blue');
+    var drawingDef= makeDrawingDef('red');
```

Comment:
Do we want to do `red`.  We use `red` for a lot of layers and we already take heat about using it.